.github/workflows/aws-ci.yaml

```diff
@@ -0,0 +1,107 @@
+name: CI tests AWS
```

Comment:
I don't see where you check for trusted or member is the author of the label ok-to-test >